homeassistant/helpers/config_entry_oauth2_flow.py

```diff
@@ -372,13 +377,18 @@ class OAuth2AuthorizeCallbackView(HomeAssistantView):
         if state is None:
             return web.Response(text=f"Invalid state")
```

Comment:
```suggestion
            return web.Response(text="Invalid state")
```

Reply:
Prefer not to change cosmetic things that is already in master. But i'll keep this suggestion open until merge, then i can add it right before merge.